src/back/dot/mod.rs

```diff
@@ -1,8 +1,9 @@
-/*! GraphViz (DOT) backend
- *
- * This backend writes a graph in the DOT format, for the ease
- * of IR inspection and debugging.
-!*/
+//! Backend for [DOT][dot] (Graphviz).
```

Comment:
Why do we need to convert the module documentation into per-line format?
I see that the current rendering is broken because of "*" at the start of lines. Just removing the "*" appears to be enough, and a single multi-line block is much easier to maintain than a bunch of individual prefixed lines.

Reply:
My reasoning was:
1. The formatting error you mentioned, which could be fixed the way you said,
2. The comment format is inconsistent between front/backends ([block comment for MSL](https://github.com/gfx-rs/naga/blob/f51f468a47312608e8f4fc59c8f1e65ce37012de/src/back/msl/mod.rs#L1) vs [line comments for GLSL](https://github.com/gfx-rs/naga/blob/f51f468a47312608e8f4fc59c8f1e65ce37012de/src/back/glsl/mod.rs)), and
3. [In-module comments use the line format.](https://github.com/gfx-rs/naga/blob/f51f468a47312608e8f4fc59c8f1e65ce37012de/src/lib.rs#L236)

If this isn't convincing, let me know and I can change them all to block comments!

Reply:
Thanks for writing all this out. I switched all of the module line comments to block comments. 😃